README.md

```diff
@@ -3,100 +3,56 @@ OpenRefine.org website
 
 We use [Docusaurus](https://docusaurus.io/) for our website.
 
-If you want to run it locally, you can follow the instructions below, otherwise you should be able to edit most pages directly with a Markdown editor.
-Opening a pull request for your changes will generate a preview that you can browse to check the rendering.
+If you want to run it locally, you can follow the instructions below, otherwise you should be able to edit most pages directly with a Markdown editor or IDE.
+Opening a pull request for your changes will generate a preview on Netlify that you can browse to check the rendering.
 
 ### Requirements
 
-Assuming you have [Node.js (LTS recommended)](https://nodejs.org/en/download/) installed (which now includes corepack), you can install Docusaurus with:
-
-Enable Corepack, if it isn't already; this will add the [Yarn](https://yarnpkg.com/getting-started/install) binary to your PATH:
-
-```sh
-corepack enable
-```
-
-Update Yarn to the latest version:
-
-```sh
-yarn set version stable
-```
+- [Node.js (LTS recommended)](https://nodejs.org/en/download/)
+  - includes npm package manager
 
 ### Installation
 
-Once you have installed yarn, navigate to docs directory & set-up the dependencies.
+Once you have installed Node, which includes the npm package manager, we can install and set-up the dependencies:
 
 ```sh
-cd docs
-yarn
+npm install
 ```
 
 ### Local Development
 
-```sh
-yarn start
-```
-
-This command starts a local development server and opens up a browser window. Usually at the URL <http://localhost:3000>
-Most changes are reflected live without having to restart the server.
-
-If you get an error starting yarn mentioning `update.latest` such as
-
-```sh
->yarn start
-
-if (notifier.update && semver.gt(this.update.latest, this.update.current)) {
-TypeError: Cannot read property 'latest' of undefined
-    at Object.<anonymous> (E:\GitHubRepos\OpenRefine\docs\node_modules\@docusaurus\core\bin\docusaurus.js:49:46)
-error Command failed with exit code 1.
-```
+The following command starts a local development server and opens up a browser window. Usually at the URL <http://localhost:3000>.
 
-then it is likely that you will need to first upgrade dependencies across all workspaces at the same time
+Most changes you make to pages and content are reflected live without having to restart the server.
 
 ```sh
-yarn up
+npm run start
 ```
 
-which will update all dependencies and store them in the `yarn.lock` file to also be committed.
-
-### Next version of OpenRefine docs
```

Comment:
@wetneb @ostephens  FYI, since we don't do versioning of docs any longer, then this section needed to be removed as well.